Yousif-Shaheen-submission/caesar.cpp

```diff
@@ -0,0 +1,83 @@
+#include <iostream>
+#include <string>
+#include <cctype>  
+
+
+const char alphabet_upper[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+const char alphabet_lower[] = "abcdefghijklmnopqrstuvwxyz";
+
+
+std::string encrypt(const std::string& word, int shift = 3) {
+    std::string final = "";
```

Comment:
I'd suggest changing it from `final`, since it's a reserved keyword in C++
```suggestion
    std::string final_string = "";
```